build.sc

```diff
@@ -72,11 +71,26 @@ trait CommonTest extends ScalaModule with TestModule.Munit {
 }
 
 object config extends Cross[ConfigModule](scalaVersions)
+
 trait ConfigModule extends CrossPlatform {
   trait Shared
       extends CrossPlatformCrossScalaModule
       with Common
-      with CommonPublish
+      with CommonPublish {
+
+    // note: somehow, this doesn't work (causes "Please override mimaPreviousVersions or mimaPreviousArtifacts")
```

Comment:
thought: I didn't spend too much time on it, hoping someone with more Mill experience can provide a better understanding and maybe a proper fix. This seems like something that only affects us until we publish the 3.x support.